examples/alerts/alerts.md

```diff
@@ -0,0 +1,143 @@
+# Alerts
+
+Here are some example alerts configured for Kubernetes environment.
+
+## Compaction
+
+```
+- alert: ThanosCompactHalted
+  expr: thanos_compactor_halted{app="thanos-compact"} == 1
+  for: 5m
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos compaction has failed to run and now is halted
+    impact: Long term storage queries will be slower 
+    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
+    dashboard: COMPACTION_URL
+- alert: ThanosCompactCompactionsFailed
+  expr: rate(prometheus_tsdb_compactions_failed_total{app="thanos-compact"}[5m]) > 0
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Compact is failing compaction
+    impact: Long term storage queries will be slower 
+    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
+    dashboard: COMPACTION_URL
+- alert: ThanosCompactBucketOperationsFailed
+  expr: rate(thanos_objstore_bucket_operation_failures_total{app="thanos-compact"}[5m]) > 0
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Compact bucket operations are failing
+    impact: Long term storage queries will be slower 
+    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
+    dashboard: COMPACTION_URL
+```
+
+## Ruler
+
+For Thanos ruler we run alerts some alerts in local Prometheus:
+
+```
+- alert: ThanosRuleIsDown
+  expr: up{app="thanos-rule"} == 0 or absent(up{app="thanos-rule"})
+  for: 5m
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Rule is down
+    impact: Alerts are not working
+    action: 'check {{ $labels.kubernetes_pod_name }} pod in {{ $labels.kubernetes_namespace}} namespace'
+    dashboard: RULE_DASHBOARD
+- alert: ThanosRuleIsDroppingAlerts
+  expr: rate(thanos_alert_queue_alerts_dropped_total{app="thanos-rule"}[5m]) > 0
+  for: 5m
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Rule is dropping alerts
+    impact: Alerts are not working
+    action: 'check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace'
+    dashboard: RULE_DASHBOARD
```

Comment:
We might want to check eval rule failures as well, but it is minor, we can add this in next PRs

Reply:
I'm not sure which metric shows that, could you point me to it?